examples/kt_android_local_test/README.md

```diff
@@ -0,0 +1,12 @@
+# Robolectric test example using kt_android_local_test
+
+To run the Robolectric test in this workspace:
+
+```
+$ bazel test //src/test:main_activity_test
+```
+
+Roboelectric tests in Bazel requires special setup to generate a file called
+`robolectric-deps.properties`, containing information about android-all jars and
+their locations. You can find this setup in the
+[bazel/robolectric](bazel/roboelectric/) directory.
```

Comment:
Where is this?

Reply:
The README.md is copied from android_local_test module. Looks like `robolectric-deps.properties` doesn't exist in sample. What about sending a new PR to remove them from those local test modules? I can do it at this week. If you prefer to it is removed at this PR, I can update new commit for it later this week.

Reply:
I think it's referring to the bazel directory in the upstream robolectric repo: https://github.com/robolectric/robolectric-bazel/blob/6d18a0ea372991da97e46b6d541dff3dfacb61b7/bazel/BUILD#L22

Can you update the text here and in the android_local_test directory to reflect that?

Reply:
Sure. I will update it this weekend.

Reply:
@jin I updated link of `robolectric-deps.properties` of `README.md` with new commit.